### ISSUE TITLE
About multi step inner loop update

### ISSUE THREAD

**Issue by GongXinyuu** [tristandeleu/pytorch-meta#45]
Hi, thanks for your great work! 
Could you please tell me how to do the multi-step gradient descent during the inner loop update?

**Comment by tristandeleu** [comment]
Hi! Thanks for the kind words, and sorry for the late reply.
The MAML example in this repo is meant to be a simple example, so this is limited to a single inner loop update. If you want to have multiple gradient steps for the inner loop with Torchmeta, I suggest you to take a look at [https://github.com/tristandeleu/pytorch-maml](https://github.com/tristandeleu/pytorch-maml), which is a better tested implementation of MAML, and more specifically [here](https://github.com/tristandeleu/pytorch-maml/blob/3c0ab81eb83686cce3109324940dbb74263bd60b/maml/utils.py#L6-L49) for how to do multiple inner loop updates.
